### PR TITLE
Update blog-guidelines.md

### PR DESCRIPTION
**Pull request by fykaa**
Rectified a minor error, and now I believe typographical errors cease to exist in this doc

### CODE REVIEW

**Comment by imnmo** [comment]
@fsmunoz @kaslin

File: communication/contributor-comms/storytelling-resources/blog-guidelines.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ This process is initiated in [#sig-contribex-comms](https://kubernetes.slack.com
...
-4. Once you have reflected any feedback in the proposal draft,  announce that the article is ready for submission (again, in the channel or in one of the weekly meetings): the assigned editor will use the final text to open the PR, adding you as a Co-author
-5. (Optional) If the submission will be mirrored in the main Kubernetes site, a second PR will be opened by the editor, but on the main repository, and after being approved in the `contributor-site` one.
+4. Once you have reflected any feedback in the proposal draft,  announce that the article is ready for submission (again, in the channel or in one of the weekly meetings): the assigned editor will use the final text to open the PR, adding you as a Co-author.
+5. (Optional) If the submission will be mirrored in the main Kubernetes site, a second PR will be opened by the editor, but on the main repository, and after being approved in the `contributor-site` once.
```
Comment (fsmunoz) [inline communication/contributor-comms/storytelling-resources/blog-guidelines.md:33]: This is actually "one", not "once": it refers to the repository.

**Review by fsmunoz — COMMENTED** [review]
All look fine, except one. If you can revert that change we can merge the correction of the typos

**Pull request merged**